src/Microsoft.DotNet.Wpf/src/Themes/PresentationFramework.Win11/Styles/RichTextBox.xaml

```diff
@@ -0,0 +1,84 @@
+﻿<!--    
+    Based on Microsoft XAML for Win UI
+    Copyright (c) Microsoft Corporation. All Rights Reserved.
+-->
+
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:controls="clr-namespace:System.Windows;assembly=PresentationFramework">
+
+    <Style x:Key="DefaultRichTextBoxStyle" TargetType="{x:Type RichTextBox}">
+        <Setter Property="Foreground" Value="{DynamicResource TextControlForeground}" />
```

Comment:
Can we use StaticResource here?

Reply:
Resolving this for now as the issue is listed. Will keep things updating there,
Issue available [here](https://github.com/dotnet/wpf/issues/8599)